doc/user/layouts/shortcodes/self-managed/troubleshoot-console-mz_catalog_server_blurb.md

```diff
@@ -0,0 +1,5 @@
+If you experience long loading screens or unresponsiveness in the Materialize
+Console, we recommend increasing the size of the `mz_catalog_server` cluster.
+Refer to the [Troubleshooting Console
+Unresponsiveness](/installation/troubleshooting/#troubleshooting-console-unresponsiveness)
+guide.
```

Comment:
the self-managed-docs/v25.1 branch uses as its basURL .../docs/self-managed/v25.1
As such, when linking to pages in our docs/user/content/path/to/pages, you only need the /path/to/pages instead of /self-managed/....

Reply:
Ahh I see. Thanks for the fix!